src/Model/Traits/CartValidatorTrait.php

```diff
@@ -27,7 +28,7 @@ public function isAmountAvailableAttribute($isStockProduct, $stockManagementEnab
     {
         $result = true;
         $unitNameString = ($unitName != '') ? ' ' . $unitName : '';
-        if ((($isStockProduct && $stockManagementEnabled) || !$alwaysAvailable) && $availableQuantity < $amount) {
+        if (((($isStockProduct && $stockManagementEnabled) || !$alwaysAvailable) && $availableQuantity < $amount) && (!Configure::read('app.selfServiceIgnoreAmountCheck') || !((new OrderCustomerService())->isSelfServiceMode()))) {
```

Comment:
die neuen conditions weiter oben in eigenem if statement und wenn dann ein return $result (das ja auf true gesetzt ist) - sonst ist das eine ewig lange wurscht